CHANGELOG.md

```diff
@@ -869,6 +891,9 @@ Other minor additions
   _⊙?_ : Decidable P → Decidable Q → Decidable (P ⟨⊙⟩ Q)
   _⊎?_ : Decidable P → Decidable Q → Decidable (P ⟨⊎⟩ Q)
   _~?  : Decidable P → Decidable (P ~)
+
+  U?   : Decidable U
+  ∅?   : Decidable ∅
   ```
```

Comment:
Typo: we shouldn't add anything to the already released 0.16.

Reply:
This is something that has been bothering for quite a while: we should probably have a

Reply:
Eek yes. I originally started this PR in v0.16, and hadn't noticed this was still floating there.

A directory of for past CHANGELOGs is a good idea. The single monolithic file is getting a little unwieldy.